lib/Push.php

```diff
@@ -379,11 +379,20 @@ protected function sendNotificationsToProxies(): void {
 		$client = $this->clientService->newClient();
 		foreach ($pushNotifications as $proxyServer => $notifications) {
 			try {
-				$response = $client->post($proxyServer . '/notifications', [
+				$requestData = [
 					'body' => [
 						'notifications' => $notifications,
 					],
-				]);
+				];
+
+				if ($proxyServer === 'https://push-notifications.nextcloud.com') {
```

Comment:
should we rtrim slashes on `$proxyServer` just in case ?

Reply:
Already trimmed here: https://github.com/nextcloud/notifications/blob/49fb541f2729994c0871b62b47143cfd0a4d8201/lib/Push.php#L347